base/gmp.jl

```diff
@@ -396,9 +396,73 @@ function (::Type{T})(n::BigInt, ::RoundingMode{:Nearest}) where T<:CdoubleMax
     x
 end
 
-Float64(n::BigInt) = Float64(n, RoundNearest)
-Float32(n::BigInt) = Float32(n, RoundNearest)
-Float16(n::BigInt) = Float16(n, RoundNearest)
+function Float64(x::BigInt)
```

Comment:
```suggestion
function Float64(x::BigInt, ::RoundingMode{:Nearest})
```
and similar for `Float32`/`Float16`.